### PR TITLE
Add CoC and Reference to wgpu Meetings

### PR DESCRIPTION
**Pull request by cwfitzgerald**
Steal bevy's COC wholesale.

Each commit is standalone.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+2. Remarks that maintainers find inappropriate, whether listed in the code of conduct or not, are also not allowed.
+3. Maintainers will first respond to such remarks with a warning.
+4. If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
+5. If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
+6. Maintainers may choose at their discretion to un-ban the user if it was a first offense and they offer the offended party a genuine apology.
+7. If a maintainer bans someone and you think it was unjustified, please take it up with that maintainer, or with a different maintainer, in private. Complaints about bans in-channel are not allowed.
+8. Maintainers are held to a higher standard than other community members. If a maintainer creates an inappropriate situation, they should expect less leeway than others.
+
+The enforcement policies in the code of conduct apply to all official venues, including Discord channels, GitHub repositories, the Twitter/Mastodon/Bluesky community and all other forums.
```
Comment (Wumpf) [inline CODE_OF_CONDUCT.md:39]: wondering if that part needs adjustment. we practically only have the google meet, the google doc, Matrix chat, and Github as official venues

**Review by Wumpf — APPROVED** [review]
thanks 👍

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (cwfitzgerald) [reply CODE_OF_CONDUCT.md:39]: I think part of the idea is to allow enforcement of this in reddit/twitdonsky  even if they aren't run by us.

Comment (ErichDonGubler) [reply CODE_OF_CONDUCT.md:39]: > twitdonsky

🤣

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ The WGPU project has multiple official platforms for community engagement:
...
   particularly useful for:
 
-  - …saying hello, and introducing yourself.
-  - …validating contributions (i.e., determining if they'll be accepted,
+  - Saying hello, and introducing yourself.
+  - Validating contributions (i.e., determining if they'll be accepted,
     ensuring your approach is correct, making sure you aren't wasting effort,
     etc.).
-  - …setting expectations for contributions.
+  - Setting expectations for contributions.
```
Comment (ErichDonGubler) [inline CONTRIBUTING.md:23]: bruh 😭

Not blocking, but this formatting _was_ on purpose.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -41,12 +41,23 @@ The WGPU project has multiple official platforms for community engagement:
...
+- `wgpu` Maintainership Meetings: Every week, the maintainership of the wgpu
+  project meets to discuss the project's direction and review ongoing work.
+  These meetings are open to the public, and you are welcome to attend. They
+  happen on Google Meet and happen on Wednesday at 16:00 UTC and last approximately
+  an hour. Remember to obey the [`CODE_OF_CONDUCT.md`] in the meeting.
+  - [Meeting Notes]
+  - [Meeting Link]
 - [GitHub discussions]: TODO: Experimentally used by some enthusiastic members
   of our community. Not supported officially.
+
```
Comment (ErichDonGubler) [inline CONTRIBUTING.md:53]: nitpick: Extra newline here.

Comment (cwfitzgerald) [reply CONTRIBUTING.md:53]: Straight to jail

**Pull request merged**